### PR TITLE
Module 1 Week 1: Review your project from trials!

### PR DESCRIPTION
**Pull request by the-catalystmc**
In this project, the following were done:

Added the .gitignore and linter files.
Added the index.html and style.css files.
Added a ReadMe file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <p>Mandela was arrested in a countrywide police swoop on 5 December 1956, which led to the 1956 Treason Trial.</p>
+        <p>On 10 May 1994 he was inaugurated as South Africa’s first democratically elected President. 
+        <p>On his 80th birthday in 1998 he married Graça Machel, his third wife. </p>
+
+        <p>True to his promise, Mandela stepped down in 1999 after one term as President</p> 
+        <p>He continued to work with the Nelson Mandela Children’s Fund he set up in 1995 and established the Nelson Mandela Foundation
+           and the Mandela Rhodes Foundation.</p>
+           <a id="tribute-link" target="_blank" href="https://en.wikipedia.org/wiki/Nelson_Mandela">Read More on Nelson Mandela Here</a>
+    </div>
+</body>
```
Comment (the-catalystmc) [inline index.html]: I would recommend using semantic tags in order to increase accessibility.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+</head>
+<body>
+    <div id = "main">
+        <h1 id = "title">Nelson Mandela</h1>
+    </div>
+    <div id="img-div">
+        <img id="image" src="https://bit.ly/3fKov2z" 
+        alt="Nelson Mandela Photo" class="responsive">
+        <p id = "img-caption">Nelson Mandela was a social rights activist, politician and philanthropist,
```
Comment (the-catalystmc) [inline index.html:17]: I would recommend using the class selector for attributes instead of the id selector.

**Review by the-catalystmc — COMMENTED** [review]
Overall a great job writing your code without any errors. 
Kindly review the tags, consider using semantic tags.
The class selector is a bit more flexible than the id selector for attributes. Kindly review using the class selector.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,28 @@
-# microverse-trial-project-review
-Add a previously completed microverse project in order to review it.
+## Microverse Trial Project Review
+A tribute page dedicated to Nelson Mandela. It was built using HTML and CSS. It is used to showcase the life and works of Nelson Mandela.
+
+#About This Project
+
+This project is created for the Microverse trials and is intended to show the experince level of HTML and CSS. The project was made using the HTML and CSS languages. These technologies were used because the project was a webpage meant to show pictures and text.
```
Comment (euqueme) [inline README.md:6]: [OPTIONAL] The objective of this project was to perform a code review for yourself on a past project, let's include that in this section.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,28 @@
...
+
+## Getting Started
+There are no installations required. Simply clone or download the zip files and open the index.html to open the webpage.
+
+## Authors
+### Author 1
+GitHub: @the-catalyst
+
+### Author 2 
+Github: @charlson91
```
Comment (euqueme) [inline README.md:19]: [OPTIONAL] You were the only author of this project so you should remove Author 2 from the README file, but if you want to give recognition I suggest creating an Acknowledgement section where you mention him.

**Review by euqueme — APPROVED** [review]
Hi @the-catalystmc  :balloon: 

Awesome job! :ok_hand: :partying_face:

You have some Code Reviewer blood in you :drop_of_blood:  you've given feedback very well  :medal_sports: it was polite and everything requested was on point :ok_hand: :relieved:  and I think you should seriously consider applying to be a Code Reviewer in the future :man_technologist:  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- This was actually in the project description but I won't stop you because of this: let's edit your review and use a couple of emojis and/or gifs in the review.
> They make the code review more human-friendly :alien: :robot:

- Let's fix your README file a bit:
> see the comments below :point_down: .

That's it! :raised_hands: 

Please :pray: do the changes :technologist: (or not) and move on to the next project :runner:  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

#### PROJECT STATUS: APPROVED :heavy_check_mark:

**Pull request merged**